src/Controllers/ResourcesCategoriesController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * @author Nicolas CARPi <nico-git@deltablot.email>
+ * @copyright 2023 Nicolas CARPi
```

Comment:
```suggestion
 * @copyright 2025 Nicolas CARPi
```
Not sure it's important but could prove to be years later :rofl: